helm_deploy/offender-events-ui/values.yaml

```diff
@@ -10,11 +10,7 @@ generic-service:
 
   ingress:
     enabled: true
-    annotations:
```

Comment:
can't see that any of these annotations are needed as are in the generic service